### ISSUE TITLE
Accessibility issue with blue-ish font

### ISSUE THREAD

**Issue by deswhite6** [LeahW/NewDevATL#19]
The blue/white color combo for the "more info" links, the Home tab selector, and the NewDevATL github link at the bottom fails WCAG guidelines for creating a site accessible to visually impaired users. WCAG contrast checker can be found here:
http://webaim.org/resources/contrastchecker/

**Comment by LeahW** [comment]
Made the blues that were too light to pass a little darker. Let me know what you think!

**Comment by GitsAndGlamour** [comment]
How do we change the hover to match the new blue?

**Comment by deswhite6** [comment]
New blue passes! Looks like the new blue affects the hover of the NewDevATL github on bottom, but not its normal color as well

**Comment by LeahW** [comment]
I actually darkened that one until it said it passed, but wanted to keep it looking different from the hover color. Does the dark teal that you see pass?

**Comment by GitsAndGlamour** [comment]
I'm going to pull request one last edit. 

I can't figure out how to change the hover-over color of the .active class to compliment the new blue so the new blue color isn't working well (unless we can change the hover color). I changed the colors to a darker teal and tested them, they pass and still compliment the teal hover color so there's less work to do.

**Comment by LeahW** [comment]
Ok, sorry for the delay there were merge errors because I pushed some fixes to the colors last night so the additions/deletions didn't match up. Just got it to go through, have a look: http://leahw.github.io/NewDevATL/
